server/controllers/controller.js

```diff
@@ -1,6 +1,41 @@
 const Model = require('../db-models/database');
 const axios = require('axios');
 
+exports.createGallery = async ({ headerImage, mainImages, moreLikeThisImages }) => {
+  let count = await Model.countDocuments({}, function (err, count) {
+    if (err) {
+        console.log(err);
+    } else {
+        return count++;
+    }
+  });
+
+  count++;
+
+  const gallery = {
+    page: count.toString(),
```

Comment:
Might be more convenient to define the page value as a Number rather than a string?

Reply:
yeah i agree with that. the original code has it stored as a string. i initially decided to go this route rather than change the seeding script and potentially create other errors.  i'll make a ticket to update while i work on the seeding script this week.